release-notes/6.0/known-issues.md

```diff
@@ -45,30 +45,29 @@ If you build .NET 6 projects with MSBuild 16.11, for example, you will see the f
 
 You can use the .net 6 SDK to target downlevel runtimes in 16.11.
 
-### 1. Windows admin installs of the .NET 6.0.400 SDK will not correctly update .NET SDK optional workloads
+### 1. Repair or update of VS or old .NET versions can lead to bad `PATH` configuration on windows
 
-Commands like `dotnet workload install` and `dotnet workload update` will not correctly update to the latest versions of the workloads on the first try. This is because a timing issue in the workload manifest update code causes the SDK to stop early typically only updating 0-1 manifests.
+In 2021, .NET Runtime and .NET SDK were changed to no longer add the `program files (x86)\dotnet` location to the global machine `PATH` but not all prior versions were updated. Some customers may still have an old version of 2.1 or older runtime and any repair of Visual Studio or that SDK will set the x86 path. Customers can end up in a state where the x86 path ends up ahead of the x64 `program files\dotnet` path which Visual Studio should be using.
 
-#### Workarounds
-
-1. Run `dotnet workload update` again and again until all workloads are updated.
-2. Run `dotnet workload update --from-rollback-file` specifying the exact workload versions you want to install.
+**Behavior**
+`The SDK 'Microsoft.NET.Sdk' specified could not be found`
 
-Example rollback file for 6.0.400
+```
+dotnet --info
 
-```console
-{
-  "microsoft.net.sdk.android": "32.0.448/6.0.400",
-  "microsoft.net.sdk.ios": "15.4.447/6.0.400",
-  "microsoft.net.sdk.maccatalyst": "15.4.447/6.0.400",
-  "microsoft.net.sdk.macos": "12.3.447/6.0.400",
-  "microsoft.net.sdk.maui": "6.0.486/6.0.400",
-  "microsoft.net.sdk.tvos": "15.4.447/6.0.400",
-  "microsoft.net.workload.mono.toolchain": "6.0.8/6.0.300",
-  "microsoft.net.workload.emscripten": "6.0.4/6.0.300"
-}
+Host:
+  Version:      7.0.5
+  Architecture: **x86**
+  Commit:       8042d61b17
 ```
 
+#### Workarounds
```

Comment:
wondering if adding screenshots would help or maybe a link to https://github.com/dotnet/core/issues/5962#issuecomment-780084535? we usually ask folks to confirm they have both by running the `where.exe dotnet` command and running those steps once we confirm x86 is first.

Reply:
That might be helpful, especially when folks hit this issue from inside VS. I think we have a screenshot from one of the error dialogs in a feedback ticket.

Reply:
Added a screenshot of the issue and linked to that comment as well. Good ideas.